db/src/test/java/org/odk/collect/db/sqlite/SqlQueryTest.kt

```diff
@@ -8,24 +8,24 @@ import org.odk.collect.shared.Query
 class SqlQueryTest {
```

Comment:
I'm thinking that this test should really be more of an "integration" one and check queries against a temp SQLite DB. That's a change we can follow up with.

Reply:
`EntitiesRepositoryTest` does that, and `SqlQueryTest` verifies only converting queries to sqlite.